pkgs/development/python-modules/pysim/default.nix

```diff
@@ -22,7 +22,7 @@
 
 buildPythonPackage {
   pname = "pysim";
-  version = "unstable-2023-08-13";
+  version = "1.0";
```

Comment:
This is wrong, 289fd28091e046868c64d29658db45f88e24900f got tagged as v1.0, not 09ff0e2b433b7143d5b40b4494744569b805e554.

Reply:
It is also older than what we have, as that got tagged in 2021.